src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(num) {
+  let sum = num || 0;
```

Comment:
The `sum` variable should be initialized to 0 directly. There's no need to use the `num` parameter or the logical OR operator here since the task specifies that the sum should start at 0 and no default value is needed for the `adder` function argument.